packages/next/client/page-loader.ts

```diff
@@ -75,23 +75,18 @@ export default class PageLoader {
       if (window.__DEV_PAGES_MANIFEST) {
         return window.__DEV_PAGES_MANIFEST.pages
       } else {
-        if (!this.promisedDevPagesManifest) {
-          // TODO: Decide what should happen when fetching fails instead of asserting
-          // @ts-ignore
```

Comment:
Psst - may I suggest using [`@ts-expect-error`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#-ts-expect-error-comments) instead? I would be very happy to send a PR migrating over! 🙂 

More broadly: I'd also suggest enabling typescript-eslint's [`prefer-ts-expect-error` rule](https://typescript-eslint.io/rules/prefer-ts-expect-error) and, even more broadly, much more of typescript-eslint's rules.

Reply:
Definitely open to migrating this over to `ts-expect-error` and investigating better TypeScript lint checks!

Reply:
Super! I'll file a GitHub issue.

For investigating more lint checks, I'd been chatting with @elsigh to try to get a more formal effort set up. Will check back in tomorrow!

Reply:
Super! For investigating more lint checks, I'd been chatting with @elsigh to try to get a more formal effort set up. Will check back in tomorrow!